### PR TITLE
feat: add How is work section

### PR DESCRIPTION
**Pull request by Elyzsium**
**Description of PR**

_A brief description of what changes your PR introduces. The better this description, the quicker we can review, exchange
feedback, and merge!_

---

Please check all that apply.

how is work section 3

- [x] Rebased with master
- [x] PR Description provided
- [ ] Did you communicate these changes with the affected teams?
- [ ] Did unit tests/integration tests/E2E tests get properly added/updated?
- [ ] Is this change production-ready including testing, feature toggles, configurations, and deployment considerations?

#### Frontend Checklist:

- [ ] Is this responsive and mobile friendly?
- [ ] Did you follow the [ARIA specs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA)

#### Backend Checklist:

- [ ] Does the endpoint return the correct success/error format?

---
**Screenshots**  
how is work

---
**Test Coverage**  
_add relevant screenshots here_

### CODE REVIEW

**Review by pelzman — DISMISSED** [review]

File: client/src/pages/Landing/HowIsWork/HowIsWork.module.scss [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    .cardGrid {
+      display: flex;
+      flex-direction: row;
+      gap: 0.312rem;
+
+      .circleWrapper {
+        position: relative;
+        width: 1.9rem;
+        height: 1.9rem;
+      }
```
Comment (KristofaJosh) [inline client/src/pages/Landing/HowIsWork/HowIsWork.module.scss:29]: looks like an unrelated style, should not be nested. did you mean differently

**Review by KristofaJosh — DISMISSED** [review]
